trk/src/Screens/SetDetail/index.js

```diff
@@ -55,33 +59,79 @@ function SetDetail(props) {
       });
   }, []);
 
+
+
+  useEffect(() => {
+    const fetchComments = async () => {
+      try {
+        const querySnapshot = await getCommentsBySomeField('climb', climbData.id);
+        const comments = [];
+        querySnapshot.forEach(doc => {
+          const commentData = doc.data();
+          comments.push({
+            id: doc.id,
+            explanation: commentData.explanation,
+            rating: commentData.rating  // Extracting the rating field
+          });
+        });
+        setFeedbackList(comments);
+      } catch (error) {
+        console.error('Error fetching comments:', error);
+      }
+    };
+    fetchComments();
+  }, [climbData.id]);
+
+  renderRating = (rating) => {
+    return '⭐️'.repeat(rating);
+  }
+
+
   return (
-    <View style={styles.container}>
-    <View style={[styles.wrapper]} >
-      <SafeAreaView />
-      <View style={styles.top}>
-        <View style={styles.topLeft}>
-          <View style={styles.gradeCircle}>
-            <Text style={styles.grade}>{climbData.grade}</Text>
+    <ScrollView>
+      <View style={styles.container}>
+        <View style={[styles.wrapper]} >
+          <SafeAreaView />
+          <View style={styles.top}>
+            <View style={styles.topLeft}>
+              <View style={styles.gradeCircle}>
+                <Text style={styles.grade}>{climbData.grade}</Text>
+              </View>
+
+              <Text style={styles.titleText}>{climbData.name}</Text>
+            </View>
+            <View style={styles.circleWrapper}>
+              <View style={styles.setterCircle}>
+                {setterImageUrl && <Image source={{ uri: setterImageUrl }} style={{ width: '100%', height: '100%' }} />}
+              </View>
+            </View>
+          </View>
+          <View style={styles.countBox}>
+            <Text style={styles.count}>{tapCount}</Text>
+          </View>
+          <View style={styles.climbPhoto}>
+            {climbImageUrl && <Image source={{ uri: climbImageUrl }} style={{ width: '100%', height: '100%' }} />}
+          </View>
+
+          <View style={styles.contentArea}>
+            {feedbackList.length > 0 && (
+              <Text style={styles.feedbackTitle}>Feedback</Text>
+            )}
+            {feedbackList.length > 0 ? (
+              feedbackList.map((comment, index) => (
+                <View key={index} style={styles.commentContainer}>
+                  <Text style={styles.commentText}>{comment.explanation}</Text>
+                  <Text style={styles.commentRating}>{renderRating(comment.rating)}</Text>
+                </View>
+              ))
+            ) : (
+              <Text style={styles.noFeedback}>No feedback yet.</Text>
+            )}
```

Comment:
A new view which renders the feedback explanation and rating. If there is no feedback associated with this climb, 'No feedback yet.' text is rendered.